Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -1463,7 +1511,84 @@ def is_ip_valid(s, accept_v6_ips=False):
         return True
 
 
-def return_outputs(readable_output, outputs=None, raw_response=None):
+class DBotScoreEntry:
+    CONTEXT_PATH = 'DBotScore'
+
+    def __init__(self, indicator, indicator_type: IndicatorType, vendor, score: DBotScore):
+        self.indicator = indicator
+        self.indicator_type = indicator_type,
+        self.vendor = vendor
+        self.score = score
+
+    def to_context(self):
+        return {
+            'Indicator': self.indicator,
+            'Type': self.indicator_type,
+            'Vendor': self.vendor,
+            'Score': self.score
+        }
+
+
+class IPIndicator:
+    CONTEXT_PATH = 'IP(val.Address && val.Address == obj.Address)'
+
+    def __init__(self, vendor, ip, asn, hostname, geo_latitude, geo_longitude, geo_country, geo_description,
+                 detection_engines, positive_engines, dbot_score: DBotScore, malicious_description):
+        self.ip = ip
+        self.asn = asn
+        self.hostname = hostname
+        self.geo_latitude = geo_latitude
+        self.geo_longitude = geo_longitude
+        self.geo_country = geo_country
+        self.geo_description = geo_description
+        self.detection_engines = detection_engines
+        self.positive_engines = positive_engines
+        self.dbot_score = DBotScoreEntry(self.ip, IndicatorType.IP, vendor, dbot_score)
+
+        if dbot_score == DBotScore.BAD:
+            self.malicious = True
+            self.malicious_vendor = vendor
+            self.malicious_description = malicious_description
+
+    def to_context(self):
+        ip_context = {
+            'Address': self.ip
+        }
+
+        if self.asn:
+            ip_context['ASN'] = self.asn
+
+        if self.hostname:
+            ip_context['Hostname'] = self.hostname
+
+        if self.geo_latitude or self.geo_country or self.geo_description:
+            ip_context['Geo'] = {}
+
+            if self.geo_latitude and self.geo_longitude:
+                ip_context['Geo']['Location'] = f'{self.geo_latitude}:{self.geo_longitude}'
```

Comment:
f-string can't be used in python2
```suggestion
                ip_context['Geo']['Location'] = '{}:{}'.format(self.geo_latitude, self.geo_longitude)
```